### PR TITLE
feat: See in world modal

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR adds the see in world modal which lets users try their wearables either in the Genesis Plaza, as it was before or at an empty plot of land, which loads way faster.
Screenshot 2023-02-09 at 08 28 32

### CODE REVIEW

File: src/components/CollectionDetailPage/CollectionDetailPage.tsx [hunk 1]
```diff
@@ -130,7 +125,7 @@ export default class CollectionDetailPage extends React.PureComponent<Props, Sta
     onNavigate(locations.collectionDetail(collection!.id, CollectionType.STANDARD, { tab }))
   }
 
-  renderMisingItemPricePopup(itemType: ItemType) {
+  renderMissingItemPricePopup(itemType: ItemType) {
```
Comment (fzavalia) [inline src/components/CollectionDetailPage/CollectionDetailPage.tsx:128]: ❤️

File: src/components/JumpIn/JumpIn.tsx [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
+    const { land } = this.props
+    if (land) {
       const selection = getSelection(land)
       const [x, y] = getCenter(selection)
-      url = getLandURL(x, y)
-    } else {
-      throw new Error('You need to supply at least one of the available props')
+      const newWindow = window.open(getLandURL(x, y), '_blank,noreferrer')
+      if (newWindow) {
+        newWindow.focus()
```
Comment (fzavalia) [inline src/components/JumpIn/JumpIn.tsx:14]: Why this?

File: src/components/JumpIn/JumpIn.tsx [hunk 3]
```diff
@@ -1,27 +1,23 @@
 import * as React from 'react'
 import { Props } from './JumpIn.types'
 import { getSelection, getCenter, getExplorerURL as getLandURL } from 'modules/land/utils'
```
Comment (kevinszuchet) [inline src/components/JumpIn/JumpIn.tsx:3]: Would you move this import one row up (it wasn't written in your PR)? 🙏

File: src/components/JumpIn/JumpIn.types.ts [hunk 4]
```diff
@@ -1,4 +1,6 @@
 import { ChainId } from '@dcl/schemas'
+import { Dispatch } from 'redux'
```
Comment (kevinszuchet) [inline src/components/JumpIn/JumpIn.types.ts:2]: Would you mind moving this import one row up? 🙏

File: src/modules/collection/utils.ts [hunk 5]
```diff
@@ -41,20 +41,32 @@ export function getCollectionEditorURL(collection: Collection, items: Item[]): s
...
 
-  if (collection) {
-    URL += `&WITH_COLLECTIONS=${collection.id}`
+  if (collectionId) {
+    URL += `&WITH_COLLECTIONS=${collectionId}`
   } else if (item_ids) {
     URL += `&WITH_ITEMS=${item_ids.join(',')}`
   }
 
+  if (position) {
```
Comment (fzavalia) [inline src/modules/collection/utils.ts:66]: The previous implementation was not providing position, no it provides 0,0 in src/components/Modals/SeeInWorldModal/SeeInWorldModal.tsx. That is the default position right?

File: src/components/Modals/SeeInWorldModal/SeeInWorld.types.ts [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import { ModalProps } from 'decentraland-dapps/dist/providers/ModalProvider/ModalProvider.types'
```
Comment (kevinszuchet) [inline src/components/Modals/SeeInWorldModal/SeeInWorld.types.ts:1]: Should this file be renamed to `SeeInWorldModal.types.ts`?

File: src/components/Modals/SeeInWorldModal/SeeInWorld.module.css [hunk 7]
```diff
@@ -0,0 +1,52 @@
+.content {
```
Comment (kevinszuchet) [inline src/components/Modals/SeeInWorldModal/SeeInWorld.module.css:1]: Should this file be renamed to `SeeInWorldModal.module.css`?

Comment (LautaroPetaccio) [reply src/components/Modals/SeeInWorldModal/SeeInWorld.module.css:1]: I completely forgot to rename the files 🤦 , good catch.

File: src/components/JumpIn/JumpIn.types.ts (hunk 4 above)
Comment (LautaroPetaccio) [reply src/components/JumpIn/JumpIn.types.ts:2]: Moved it!

File: src/components/JumpIn/JumpIn.tsx (hunk 2 above)
Comment (LautaroPetaccio) [reply src/components/JumpIn/JumpIn.tsx:14]: I copied the same functionality from the `JumpIn` button, but it seems that it's not required, the window gets focused automatically.

**Review by kevinszuchet — APPROVED** [review]

**Pull request merged**